rules/game_process.tex

```diff
@@ -598,3 +608,8 @@ \subsubsection{Sudden Death Shoot-Out}
   \item Otherwise, the sudden death is repeated.
   \item If after 3 sudden death penalty kicks there is still no winner, the referee will toss a coin to decide the game.
 \end{itemize}
+
+\subsection{Aborting A Penalty}
+\label{sec:aborting_penalty}
+
+If a penalty was incorrectly applied to a robot and acknowledged by the referee to be incorrect, the robot should first be removed and placed outside the field, following the same process for the Standard Removal Penalty. Once placed, the robot should then be allowed to continue play immediately after being placed and the GameController removes its penalty.
```

Comment:
In this case the penalty count for the team should not include the aborted penalty.
Furthermore, the penalty time increase described in the following text should not apply to an aborted penalty: "Unless otherwise specified, the penalty time increases by 10s each time a team commits any infraction"